addon/components/lmb-plugin/list.hbs

```diff
@@ -0,0 +1,60 @@
+<AuTable>
+  <:header>
+    <tr class='au-c-data-table__header-title'>
+      <th>
+        <WorshipPlugin::SortableTableHeader
```

Comment:
If you're reusing these, it makes sense to extract them out from under the worshipPlugin namespace

Reply:
I used Common::Sort::SortableTableHeader is this a good naming?